spec/pact-helper.js

```diff
@@ -0,0 +1,38 @@
+var request = require('request');
+var pact = require('../pact-provider-dsl');
+
+var changeState = function (status, response, callback) {
```

Comment:
Yes.

The `this.setUp` function is the actual api we will provide. Since all the setUp functions is this example make a similar http request I've created this local function to remove the duplication.